homeassistant/components/lastfm/coordinator.py

```diff
@@ -0,0 +1,35 @@
+"""Data update coordinator for the LastFM integration."""
```

Comment:
Could you implement the coordinator in a preliminary PR?

Reply:
I am having trouble getting tests to work when trying to add the coordinator to the not async way of working. What would be the best way to continue?

Reply:
#91606 is my stub

Reply:
I actually think the coordinator is not necessary at all in this circumstance.
You can do the config-flow without implementing a coordinator - and it can always be added as a follow-up PR if there is a need.